test/server_integration/http/ssl_redirect/config.ts

```diff
@@ -21,6 +22,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
   const redirectPort = httpConfig.get('servers.kibana.port') + 1234;
 
   return {
+    testConfigCategory: ScoutTestRunConfigCategory.UI_TEST,
```

Comment:
same here, probably  `API_TEST`

Reply:
There's a follow-up PR for it already: #2107